### ISSUE TITLE
issue with Spain Account

### ISSUE THREAD

**Issue by gchamorro2000** [OCA/sale-workflow#3384]
module: sale_advance_payment
version: 16.0

Location Spain
Hello, I would like to know where the payment accounting accounts are configured, since the ones shown by default are incorrect in Spain.
![image](https://github.com/user-attachments/assets/22b4f0eb-8b0e-492d-b3f3-18b4e251bf4b)
![Captura de pantalla 2024-10-31 100326](https://github.com/user-attachments/assets/5ee6da36-e8f7-4685-9e2d-b62c81f22c51)

**Comment by corredato** [comment]
@gchamorro2000 I don't think this is an issue. You can install the Spanish chart of accounts in the the apps list (l10n_es).

Please treat anything related to localization or if it has the l10n prefix in their respective repositories. (https://github.com/OCA/l10n-spain)

**Comment by corredato** [comment]
@mymage I believe we can close this
